gapic/src/main/com/google/gapid/views/PerformanceView.java

```diff
@@ -119,6 +126,13 @@ public PerformanceView(Composite parent, Models models, Widgets widgets) {
     filterButton.setImage(widgets.theme.more());
     filterButton.setLayoutData(new GridData(SWT.LEFT, SWT.CENTER, false, false));
 
+    if(ProfileExperiments.SHOULD_ENABLE_EXPERIMENTS) {
+      Button experimentsButton =createButton(buttonsComposite, SWT.FLAT, "Experiments",buttonColor,
+      e -> widgets.experiments.showExperimentsPopup(getShell()));
+      experimentsButton.setImage(widgets.theme.clipboard());
```

Comment:
Why the clipboard icon?

Reply:
It's just a placeholder. We haven't decided what to put yet and it's not visible to the user anyway. We can change when we decide what icon to put or not put at all.

Reply:
I'd prefer we pick an icon now. How about the "science" icon from material. It's a flask, I think that works.

Reply:
Sure, I will update it.